set_version.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
```

Comment:
Is this file omited in the setuptools package?

Reply:
Should be

Reply:
haha, I'll double check when I merge ;)